### PR TITLE
Add footers for '04 - Guides, Workflows, & Courses'

### PR DESCRIPTION
**Pull request by claremacrae**

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/Community Talks/Metadata Group Discussion.md [hunk 1]
```diff
@@ -8,4 +8,10 @@ publish: true
 
 # Metadata Group Discussion
 
-This [[Obsidian Community Talks|community talk]] was a group discussion about ways people use and manage their [[YAML frontmatter]]. 
\ No newline at end of file
+This [[Obsidian Community Talks|community talk]] was a group discussion about ways people use and manage their [[YAML frontmatter]].
```
Comment (dbarenholz) [inline 04 - Guides, Workflows, & Courses/Community Talks/Metadata Group Discussion.md:11]: What's happening here with the line?

File: 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md [hunk 2]
```diff
@@ -44,4 +44,10 @@ tags:
 2. Apply the [[T - New Note]] template
 3. Link the new guide in of the overview notes located in `04 - Guides, Workflows, & Courses`
 4. Write to your heart's content!
-5. [[How to add content through GitHub|Submit your changes to GitHub]]
\ No newline at end of file
+5. [[How to add content through GitHub|Submit your changes to GitHub]]
```
Comment (dbarenholz) [inline 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md:47]: Same here: what's happening with the line? Why is it deleted & added?

File: 04 - Guides, Workflows, & Courses/for Beginners.md [hunk 3]
```diff
@@ -14,4 +14,10 @@ These guides will help you get started with [[Obsidian]] and related topics.
 - [[Obsidian Help]]
 - [[Obsidian Garden]]
 - [[YouTube]]
-- [[Course for Getting Started with Obsidian]]
\ No newline at end of file
+- [[Course for Getting Started with Obsidian]]
```
Comment (dbarenholz) [inline 04 - Guides, Workflows, & Courses/for Beginners.md:17]: Deleted & added line: why?

File: 04 - Guides, Workflows, & Courses/for Academic Writing.md [hunk 4]
```diff
@@ -8,4 +8,10 @@ tags:
 
 # for Academics
 - [Comprehensive Academic Workflow from Reading to Writing in Markdown](https://www.notion.so/chrisgrieser/Comprehensive-Academic-Workflow-from-Reading-to-Writing-in-Markdown-a62298be91934043b11006be1ddc553a) by [[chrisgrieser|pseudometa]]
-- [Zotero -> zotfile -> mdnotes -> obsidian -> dataview Workflow](https://forum.obsidian.md/t/zotero-zotfile-mdnotes-obsidian-dataview-workflow/15536) by @Cat
\ No newline at end of file
+- [Zotero -> zotfile -> mdnotes -> obsidian -> dataview Workflow](https://forum.obsidian.md/t/zotero-zotfile-mdnotes-obsidian-dataview-workflow/15536) by @Cat
```
Comment (dbarenholz) [inline 04 - Guides, Workflows, & Courses/for Academic Writing.md:11]: Deleted & added line: why?

**Review by dbarenholz — APPROVED** [review]
Do look at comments!

File: 04 - Guides, Workflows, & Courses/Community Talks/Metadata Group Discussion.md (hunk 1 above)
Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/Community Talks/Metadata Group Discussion.md:11]: There was no end-of-line character in the original file.

This original meant that the `%% Hub footer: Please don't edit anything below this line %%` comment ran right up against the end of the content.

I made the script first effectively:

1. add the missing end-of-line character at the end of the file, and then
2. add the footer.

Git shows this up as an edit on the last line of the file...

**Pull request merged**